features/324-config/conventions.md

```diff
@@ -0,0 +1,56 @@
+# Configuration Conventions
+
+## Background
+
+User config is the user-facing configuration that allow user to enable and provide configurations to Skygear provided gears. This document is intended to provide conventions to ensure the configuration is clear, easy to understand, flexible for new features and consistent.
+
+## Conventions
+
+### Prefer using list of items over map
+
+To defines multiple items (e.g. login id keys, sso providers) in config, don't use map. A list of items containing the key field should be used instead.
```

Comment:
Is the field name `key` unconditional for all cases? Maybe in some cases, we may want to use other terms, say `name`, `id`, etc.

Reply:
I thought the key field is added only when necessary. Rephrase the sentence and see if it looks better.